r/_pkgdown.yml

```diff
@@ -164,6 +164,9 @@ reference:
       - match_arrow
       - value_counts
       - list_compute_functions
+  - title: Connections to other systems
+    contents:
+      - to_duckdb
```

Comment:
Scope creep: should we manufacture a man page for the r_to_py/py_to_r methods?

Reply:
We should, though I'm going to kick that can down the road with https://issues.apache.org/jira/browse/ARROW-13442